README.md

```diff
@@ -169,8 +169,49 @@ Using pgadmin4 we can inspect the postgres database's schemas and tables (helpfu
 8. You'll probably want to look at Schemas > (pick a schema) > Tables
 
 
-## Contributing
+### Enabling Google Sign In (Optional)
 
-See [CONTRIBUTING.md](https://github.com/bytedeck/bytedeck/blob/develop/CONTRIBUTING.md) if you plan to contribute code!
 
+Here are the steps, assuming that you now have a functional tenant:
+
+1. Obtain Google credentials: https://developers.google.com/workspace/guides/create-credentials#oauth-client-id
+2. Make sure that in the Authorized URIs, add `http://hackerspace.localhost.net:8000/accounts/google/login/callback/`. We will explain why we are using `localhost.net` later but for now, just add this.
+3. Go to Social Applications: http://hackerspace.localhost:8000/admin/socialaccount/socialapp/
```

Comment:
Every tenant has to do this?

Reply:
This seems like something that should be automated when a tenant owner turns the setting on in their SiteConfig.  This is not something we can rely on tenant owners (i.e. lay people) to do.

Reply:
Gotcha. As per my comment above, the solution here would be to move `allauth.socialaccount` to `SHARED_APPS` so we only need to configure it in the public admin

Reply:
Seems like we can't just move the `allauth.socialaccount` to `SHARED_APPS` only since each tenant has different users.

Instead, I did a hack that automatically copies the social app configuration from admin to tenant when they enable google sign in.